apps/marketing/content/blog/manfunction-mania.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Announcing Malfunction Mania
+description: We are getting ready to release Documenso 1.0! Join in on the fun of making sure the release is on point. We are calling MALFUNCTION MANIA.
+authorName: 'Timur Ercan'
+authorImage: '/blog/blog-author-timur.jpeg'
+authorRole: 'Co-Founder'
+date: 2023-09-20
+tags:
+  - Testing
+  - Rewrite
+  - Bounties
+---
+
+<figure>
+  <MdxNextImage
+    src="/blog/new_bug.png"
+    width="1260"
+    height="630"
+    alt="XKCD: Bug"
+  />
+
+  <figcaption className="text-center">
+    XKCD: New Bug
+  </figcaption>
+</figure>
+
+**☢️ We are calling a MALFUNCTION MANIA!**
+
+**🚨 Documenso 1.0 is in the [testing enviroment](https://documen.so/test)**
+
+It has been a minute since Lucas proclaimed [“We are doing a rewrite”](https://documenso.com/blog/why-were-doing-a-rewrite) and many of you have been asking, when the new version will be availible. I’m happy to say, that the wait has come to an end. The work on Documenso 1.0 has reached a level we feel comfortable going into the next phase with. We had a lot of community feedback, contributions and moral support to get us this far, which is why we are excited to annouce our biggest commnity project yet:
+
+As Documenso 1.0 just hit our test environment, and we are calling a “MALFUNCTION MANIA”. A big, public testing phase, where we invite everyone to try out the new version, hunt down, report and fix any and all malfunctions (aka bugs) and give feedback before release. Malfunction Mania will happen alongside our own internal testing and by combining the two, we want to ensure the best possible release we can have for Version 1.0. We know that many of you have been eager to contribute and this is your chance (the first of many to come).
+
+## As part of Malfunction Mania, we are offering special bounties:
+
+- $50 for reporting serious issues in a reproduceable way
+- $100 for fixing reported or other critical issues via accepted PR
+- Exclusive Merch: Every worthy contribution to Malfunction Mania will be awarded the Limited Edition “Signature Look” Shirt
```

Comment:
I think we should have some more nuance on this since not all bugs are the same.

Reply:
@catalinpit @dguyen @dguyen thoughts?